requirements.txt

```diff
@@ -8,5 +8,5 @@ tox
 pep8
 -e git://github.com/willthames/ansible-lint.git@9777f20aafd4be8706c9ce17005c981cb19a6a25#egg=ansible-lint
 -e git://github.com/blueboxgroup/ansible.git@stable-1.9-bb#egg=ansible
--e git://github.com/blueboxgroup/ursula-cli.git@ansible-1.9#egg=ursula-cli
+-e git://github.com/blueboxgroup/ursula-cli.git#egg=ursula-cli
```

Comment:
Why is this being changed? We want to pin to ansible 1.9 version of ursula-cli, so that when we update ursula-cli for ansible 2.0 we don't break these existing branches / tools.